tests/test_data/test_gold_translator.py

```diff
@@ -376,22 +407,25 @@ def test_get_lat_lon():
 def test_get_instrument():
     translator = GoldStudyTranslator()
 
-    instrument = translator._get_instrument(
-        {
-            "seqMethod": ["Illumina NextSeq 550", "Illumina NextSeq 3000"],
-        }
-    )
-    assert instrument.id == "nmdc:inst-14-xz5tb342"
-
-    instrument = translator._get_instrument(
-        {
-            "seqMethod": [],
-        }
-    )
-    assert instrument is None
-
-    instrument = translator._get_instrument({"seqMethod": None})
-    assert instrument is None
+    with patch.object(
+        translator, "gold_nmdc_instrument_map_df", new=mock_gold_nmdc_instrument_map_df
+    ):
```

Comment:
Is patching necessary here? Why not just pass `mock_gold_nmdc_instrument_map_df` to the `GoldStudyTranslator` constructor? That more accurately reflects how the class is used.

Reply:
@sujaypatil96

Reply:
Oh oops yes! much better than having to patch. Made a commit to do the above.